go/tools/builders/generate_test_main.go

```diff
@@ -191,14 +191,13 @@ func main() {
 	if failfast := os.Getenv("TESTBRIDGE_TEST_RUNNER_FAIL_FAST"); failfast != "" {
 		flag.Lookup("test.failfast").Value.Set("true")
 	}
-
-	// Setting this flag serves two purposes:
-	// 1. It attains parity with "go test", which enables this feature by default.
-	//    https://cs.opensource.google/go/go/+/refs/tags/go1.18.1:src/cmd/go/internal/test/test.go;l=1331-1337
-	// 2. It provides a way to run hooks right before testing.M.Run() returns.
-	flag.Lookup("test.paniconexit0").Value.Set("true")
-
-	{{if ne .CoverMode ""}}
+{{if eq .CoverFormat "lcov"}}
+	panicOnExit0Flag := flag.Lookup("test.paniconexit0").Value
+	testDeps.OriginalPanicOnExit, _ = strconv.ParseBool(panicOnExit0Flag.String())
```

Comment:
FWIW, you don't have to re-parse.
flag.Bool is backed by a [boolValue](https://cs.opensource.google/go/go/+/refs/tags/go1.18.2:src/flag/flag.go;l=126-140;drc=f12461cb0f3878b6d332f4be3a02fa44cc744df6),
which always [implements](https://cs.opensource.google/go/go/+/refs/tags/go1.18.2:src/flag/flag.go;l=140;drc=f12461cb0f3878b6d332f4be3a02fa44cc744df6) the [flag.Getter](https://pkg.go.dev/flag#Getter) interface.

So you can just upcast panicOnExit0Flag to Getter and take the parsed value out.

```suggestion
	testDeps.OriginalPanicOnExit = panicOnExit0Flag.(Getter).Get().(bool)
```